.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "none",
+  "tabWidth": 4,
+  "singleQuote": true,
+  "semi": true
```

Comment:
Could you set that to false?

Reply:
Updated in a recent commit; also re-formatted each of the files in a dedicated commit.